docs/references/i18n.rst

```diff
@@ -7,137 +7,166 @@ Internationalization
+
+
+Prerequisites
+~~~~~~~~~~~~~
+
+First, you'll need to have these dependencies available on your path:
 
-Translations are maintained in release branches on CrowdIn in the following projects:
-  
-* `CrowdIn: kolibri <http://crowdin.com/project/kolibri>`__
-* `CrowdIn: kolibri-docs <http://crowdin.com/project/kolibri-docs>`__
+* GNU ``gettext``
+* Java
 
-To build translated user documentation, please see the `kolibri-docs repository <https://github.com/learningequality/kolibri-docs/>`__.
+You may be able to install them using your system's package manager.
 
-In order to pull the latest messages, invoke the following commands, fetching your API key using `these instructions <https://support.crowdin.com/api/api-integration-setup/>`__. Notice that you have to specify from which branch you are pulling the translations.
+Next, download the crowdin jar to the current directory using this command:
 
 .. code-block:: bash
 
-    $ CROWDIN_API_KEY="your-secret-key" make translation-crowdin-download branch=release-v0.7.x
+    $ make translation-crowdin-install
 
-.. warning:: By default Crowdin will download all translations, not just approved ones, and will often download untranslated strings also. Do not just add all the files that are downloaded, as this will lead to untranslated and poor quality strings being included.
 
+Finally, ensure you have an environment variable ``CROWDIN_API_KEY`` set to the Learning Equality organization `account API key <https://support.crowdin.com/api/api-integration-setup/>`_.
 
-Fetching and building translations
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. note:: We do not currently support making translations on Windows. It might be possible, but would require inspection of the Makefile and running alternate commands.
+
+.. note:: If you install ``gettext`` on Mac with Homebrew, you may need to add the binary to your path manually
+
+
+Exporting and uploading
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Typically, strings will be uploaded when a new release branch is cut from ``develop``, signifying the beginning of string freeze and the ``beta`` releases.
+
+Before translators can begin working on the strings in our application, they need to be uploaded to Crowdin. Translations are maintained in release branches on Crowdin in the `Crowdin kolibri project <http://crowdin.com/project/kolibri>`_.
+
+This command will extract front- and backend strings from the Kolibri application code:
+
+.. code-block:: bash
 
-Each 30 minutes, you can fetch updated translations from CrowdIn. After changing and approving translations on CrowdIn, you should download them and build them to see if they work before releasing them.
+  $ make translation-extract
 
-UI (backend & frontend):
+And this command will upload the strings to Crowdin:
 
 .. code-block:: bash
 
-    $ CROWDIN_API_KEY="your-secret-key" make translation-crowdin-download branch=release-v0.7.x
-    $ make translation-django-compilemessages
```

Comment:
I see that the point of the changing of the `translation-crowdin-download` might have been to hide this step from the developer. Just wanna flag that I think it's a good idea for developers to know what's going on in these processes.

1. Download from CrowdIn
2. Run Django's `compilemessages`